packer/freebsd.pkr.hcl

```diff
@@ -9,6 +9,7 @@ variable "prefix" {
 
 locals {
   name = "${var.prefix}pg-ci"
+  image_name = "${local.name}-${var.task_name}-${var.image_date}"
```

Comment:
In other places you treat image_name as the portion without the date. Perhaps it'd be better to name the variable image_identity?